dbt/README.md

```diff
@@ -64,17 +64,22 @@ dbt_helper update-tables --help
 
 #### `validate`
 
-If you want to check if a materialized asset passes the validation tests defined in DBT, you can use `dbt_helper validate`. You can pass in the `--select` parameter to filter to a specific set of validation tests, and `--target` to set the DBT target.
+If you want to check if a materialized asset passes the validation tests defined in DBT, you can use `dbt_helper validate`.
 
-This will run the associated tests, then print out the test query output so you can see what went wrong.
+This understands how to translate dagster asset selections into DBT node selections, and does some extra legwork to make the test outputs more informative.
+
+See `dbt_helper validate --help` for usage details.
```

Comment:
Decided to mostly just leave a pointer here - `dbt_helper validate --help` is (a) closer to the code so easier to maintain and (b) closer to the terminal where people are already typing `dbt_helper validate whatever`.